samples/assistant/csharp-legacy/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    "AssistantSample": {
```

Comment:
nit: should we name this `AssistantLegacySample` just to make it clear to customers that this isn't the version of c# samples they should be using?